test/tasks/kompo_vfs_path_test.rb

```
@@ -231,6 +231,108 @@ def test_from_source_has_repo_url_constant
+
+      original_home = ENV["HOME"]
+      ENV["HOME"] = tmpdir
+
+      capture_io { Kompo::KompoVfsPath::FromGitHubRelease.run }
+
+      assert_requested(:get, url)
+      assert @mock.called?(:run, "tar")
+    ensure
+      ENV["HOME"] = original_home
+    end
+  end
```

RuboCop (1.84.1)

[convention] Assignment Branch Condition size for `test_downloads_and_extracts_tarball` is too high. [<16, 26, 1> 30.55/23]

(Metrics/AbcSize)